### ISSUE TITLE
java.lang.NullPointerException: Pointer address of argument 3 is NULL on org.bytedeco.opencv.global.opencv_imgproc.rectangle

### ISSUE THREAD

**Issue by 4714407** [bytedeco/javacv#1599]
The following code runs with an NullPointerException
```
rectangle(frame,
                                new Point(result.x, result.y),
                                new Point(result.x + result.width, result.y + result.height),
                                Scalar.MAGENTA, 2, LINE_8, 0);
```
java.lang.NullPointerException: Pointer address of argument 3 is NULL.
	at org.bytedeco.opencv.global.opencv_imgproc.rectangle(Native Method) ~[opencv-4.4.0-1.5.4.jar:4.4.0-1.5.4]

argument 3 isScalar.MAGENTA，What did I write wrong?

**Comment by saudet** [comment]
That should be fixed with commit https://github.com/bytedeco/javacpp-presets/commit/7f05222bf456d0423f2e3902d692d7d3a92cd335
Please give it a try with the snapshots: http://bytedeco.org/builds/
Thanks for reporting!

**Comment by 4714407** [comment]
This is what I write now:
```
rectangle(frame,
                                new Point(result.x, result.y),
                                new Point(result.x + result.width, result.y + result.height),
                                new Scalar(255, 0, 255, 0), 2, LINE_8, 0);
It seems that there is no problem. Should I change it to:
```
rectangle(frame,
                                new Point(result.x, result.y),
                                new Point(result.x + result.width, result.y + result.height),
                                new Scalar(255, 0, 255, 0).retainReference(), 2, LINE_8, 0);

**Comment by saudet** [comment]
Since you're allocating a new object each time, it's not going to make any
difference, no.

**Comment by 4714407** [comment]
Thank you very much.

**Comment by saudet** [comment]
The fix has been released with version 1.5.5! Enjoy
